Library/Formula/slib.rb

```diff
@@ -0,0 +1,24 @@
+class Slib < Formula
+  homepage "http://people.csail.mit.edu/jaffer/SLIB"
+  url "http://groups.csail.mit.edu/mac/ftpdir/scm/slib-3b4.zip"
+  version "3b4"
+  sha1 "dda1ed78ff7164738a1a8c51f1f7c08ec1db79eb"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "infoz", "INSTALL_INFO=install-info"
+    system "make", "install", "INSTALL_INFO=install-info"
+  end
+
+  def caveats; <<-EOS.undent
+    SLIB distribution is installed at /usr/local/lib/slib/.
+
+    See `info slib' to see how to configure your Scheme implementation to
+    locate SLIB, load SLIB initialization file and so on.
+    EOS
+  end
+
+  test do
+    system "#{bin}/slib", "-v"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!